source/administration/bulk-export-data.rst

```diff
@@ -244,6 +321,60 @@ UserChannelMembership object
       <td valign="middle">roles</td>
       <td valign="middle">string</td>
       <td>The roles the user has within this channel. </td>
+    </tr>
+        <tr class="row-odd">
+      <td valign="middle">notify_props</td>
+      <td valign="middle">object</td>
+      <td>The notify preferences for this user in this channel. </td>
```

Comment:
I would say here something similar to the User notify props, something like "as defined in ChannelNotifyProps"

Reply:
Updated.